DeskKit/DeskKit/View Models/DKContactUsViewModel.m

```diff
@@ -142,9 +162,52 @@ + (BOOL)isEmptyString:(NSString *)string
     return string == nil || [string isEqualToString:@""];
 }
 
++ (BOOL)isNotEmptyString:(NSString *)string
+{
+    return ![[self class] isEmptyString:string];
+}
```

Comment:
Yeah, I was thinking with a prefixed method name.  To your point though, having the SDK pollute the global namespace at all (even with a prefix) could be an annoyance.